torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp

```diff
@@ -68,6 +68,24 @@ static std::atomic<bool> executor_mode{true};
 static std::atomic<bool> profiling_mode{true};
 #endif
 
+std::mutex fusion_strategy_lock;
+static FusionStrategy fusion_strategy = {{FusionBehavior::STATIC, 20}};
```

Comment:
nit: would be great if we could add some documentation to python bindings that explain to a user how strategies work exactly and in which order they apply?

Reply:
My one worry is that we’re exposing implementation details… but yea I’ll add something with warnings/disclaimers at least